boards/xtensa/esp32/ttgo_lora_esp32/scripts/Make.defs

```diff
@@ -73,7 +73,7 @@ ifneq ($(CONFIG_DEBUG_NOOPT),y)
   ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing
 endif
 
-ARCHCFLAGS = -fno-builtin -ffunction-sections -fdata-sections -fstrict-volatile-bitfields -mlongcalls
+ARCHCFLAGS = -fno-builtin -ffunction-sections -fdata-sections -mlongcalls
```

Comment:
Do we need to add `ifeq ($(CONFIG_FRAME_POINTER),y)`?

Reply:
ditto